back/src/Model/GameRoom.ts

```diff
@@ -128,6 +132,25 @@ export class GameRoom implements BrothersFinder {
             mapDetails.thirdParty ?? undefined
         );
 
+        if (mapDetails.canEdit) {
```

Comment:
Replace with:
```suggestion
        if (mapDetails.editable) {
```

This is a new feature available in https://github.com/thecodingmachine/workadventure/pull/3069